app/forms.py

```diff
@@ -318,3 +319,56 @@ def validate_name_plural(form, field):
 			raise ValidationError('Another food resource type already has this \
 				plural name. Plural names must be unique.')
 
+class CheckboxForm(Form):
+	id = TextField()
+	value = BooleanField(
+		validators = []
+	)
+
+	def set_label(self, label):
+		self.value.label = Label(self.value.id, label)
+
+class PrecheckedCheckboxForm(Form):
```

Comment:
This model seems redundant. What do you think about inheriting from `CheckboxForm`? Or overriding `CheckboxForm`'s `__init__` method to take a boolean to set the default value for the `BooleanField`?